<?xml version="1.0"?>

<launch>
    <arg name="conf_threshold" default="0.1"/>
    <arg name="blue_count_threshold" default="20"/>
    <arg name="red_count_threshold" default="50"/>

    <node name="traffic_light_detector" pkg="traffic_light_detector" type="traffic_light_detector.py" cwd="node" output="screen">
        <remap from="CompressedImage" to="grasscam/image_raw/compressed"/>
        <param name="conf_threshold" value="$(arg conf_threshold)"/>
        <param name="blue_count_threshold" value="$(arg blue_count_threshold)"/>
        <param name="red_count_threshold" value="$(arg red_count_threshold)"/>
    </node>

    <!-- <node pkg="rviz" type="rviz" name="rviz" args="-d $(find traffic_light_detector)/config/config.rviz"/> -->
</launch>
